processing/data_collection/gazette/spiders/sc_florianopolis.py

```diff
@@ -0,0 +1,73 @@
+import re
+from datetime import date, datetime
+
+from dateparser import parse
+from dateutil.relativedelta import relativedelta
+from scrapy import FormRequest, Spider
+
+from gazette.items import Gazette
+
+
+class ScFlorianopolisSpider(Spider):
+    name = 'sc_florianopolis'
+    URL = 'http://www.pmf.sc.gov.br/governo/index.php?pagina=govdiariooficial'
+    MUNICIPALITY_ID = '4205407'
+    AVAILABLE_FROM = date(2009, 6, 1)
```

Comment:
How about changing this to 2015-01-01? Not sure if we're interested in handling download and parsing issues going further this year.

This is an open discussion on this issue: https://github.com/okfn-brasil/diario-oficial/issues/23